cycledash/static/js/runs/components/forms.js

```diff
@@ -93,30 +100,37 @@ var NewBAMForm = React.createClass({
   },
   render: function() {
     return (
-      <form method='POST' action='/api/bams' className='bam-form' ref='bamForm'>
-        <h3>New BAM</h3>
-
-        <TextInput label='Name:' name='name' required={true}
-                   placeholder='...' />
-        <TextInput label='Tissues:' name='tissues'
-                   placeholder='Left Ovary' />
+      <form method='POST' action='/api/bams' className='add-form' ref='bamForm'>
+        <div className='row'>
+          <h3>New BAM</h3>
+          <div className='add-form-input-half'>
```

Comment:
Good call! I removed that extra div and added the `.add-form-input-half` class to the enclosing `.form-group` instead. It's way cleaner to apply the Bootstrap grid to a wrapper div, since you need both the input and its label to sit inside the grid. And Bootstrap has some styling that goes along with their `.form-control` input class that would be messy to override. Will push soon!